tests/unit/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from west.utils import is_version
+from west.utils import sort_versions
+from west.utils import until
+
+
+def test_sort_versions():
+    values = sort_versions(["1.0.1", "2.0", "1.0.3"])
+
+    assert values == ["1.0.1", "1.0.3", "2.0"]
```

Comment:
I think we should add a case with `1.1` and `10.0`, this is a classical trap.

Reply:
fixed